pkg/products/threescale/objects_test.go

```diff
@@ -881,9 +895,11 @@ func getSuccessfullTestPreReqs(integreatlyOperatorNamespace, threeScaleInstallat
 func getValidInstallation(installationType integreatlyv1alpha1.InstallationType) *integreatlyv1alpha1.RHMI {
 	return &integreatlyv1alpha1.RHMI{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:       "test-installation",
-			Namespace:  integreatlyOperatorNamespace,
-			Finalizers: []string{"finalizer.3scale.integreatly.org"},
+			Name:            "test-installation",
+			Namespace:       integreatlyOperatorNamespace,
+			Finalizers:      []string{"finalizer.3scale.integreatly.org"},
+			Generation:      1,
+			ResourceVersion: "1",
```

Comment:
We have gone through it on call, this is not an issue since every test case get's new object. However, this would be an issue if the objects would be instantiated above test cases.